src/android/io/branch/BranchSDK.java

```diff
@@ -482,23 +518,28 @@ private BranchLinkProperties createLinkProperties(JSONObject options, JSONObject
 
     /**
      * Mark the content referred by this object as viewed. This increment the view count of the contents referred by this object.
+     *
+     * @param instanceIdx The instance index from branchObjects array
      */
-    private void registerView()
+    private void registerView(int instanceIdx)
     {
 
         Log.d(LCAT, "start registerView()");
 
-        this.branchObj.registerView();
+        BranchUniversalObject branchObj = (BranchUniversalObject)this.branchObjects.get(instanceIdx);
+
+        branchObj.registerView();
```

Comment:
This is missing the RegisterViewListener that I added in https://github.com/BranchMetrics/Cordova-Ionic-PhoneGap-Deferred-Deep-Linking-SDK/pull/74/files